xmsannotator/tests/testthat.R

```diff
@@ -1,3 +1,4 @@
+library(patrick)
```

Comment:
if we indend to test with conda we might need to wrap patrick for it

Reply:
I'm just checking on it. It's already underway: https://github.com/conda-forge/staged-recipes/pull/15896